extras/agc-minimal-permissions/package.json

```diff
@@ -11,19 +11,23 @@
     "cdk": "cdk"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.126.0",
+    "@aws-cdk/assert": "^2.50.0",
     "@types/jest": "^27.0.2",
     "@types/node": "16.10.3",
-    "aws-cdk": "^1.126.0",
+    "aws-cdk": "2.50.0",
```

Comment:
nit: should we use `^2.50.0` to get latest version?

Reply:
Updated